### ISSUE TITLE
Add new Task/Batch features into DB ATs

### ISSUE THREAD

**Issue by sabbyanandan** [spring-cloud/spring-cloud-dataflow-acceptance-tests#272]
@sabbyanandan commented on [Tue Jul 23 2019](https://github.com/spring-cloud/spring-cloud-dataflow/issues/3366)

- [ ] Task execution deletes
- [ ] Task execution stop
- [ ] Task search

**Comment by jvalkeal** [comment]
Just a note, task execution stop testing is still on hold until we can verify successful stop from api's.

**Comment by cppwfs** [comment]
@jvalkeal the task stop feature does work.  However the exit code will be zero.   In your comment above are you referring to the task returning an exit code that contains the signal that facilitated the stop?

**Comment by sabbyanandan** [comment]
Blocked by https://github.com/spring-cloud/spring-cloud-task/pull/619.
